builtin/providers/google/resource_google_project_services.go

```diff
@@ -160,7 +168,9 @@ func getApiServices(pid string, config *Config) ([]string, error) {
 		return apiServices, err
 	}
 	for _, v := range svcResp.Services {
-		apiServices = append(apiServices, v.ServiceName)
+		if _, ok := ignore[v.ServiceName]; !ok {
+			apiServices = append(apiServices, v.ServiceName)
+		}
```

Comment:
Outcome of the conversation is largely that DiffSuppressFunc and Sets/Lists is largely uncharted territory, and we don't know a ton about its behaviour just yet.

I think @danawillow was going to investigate more, but unless she surfaces information that can make this work cleanly, I think the implementation as it exists is as good as this is going to get, and am fine moving forward with it.